### ISSUE TITLE
Extras, Configure... Currency conversion not editable

### ISSUE THREAD

**Issue by ZZJHONS** [rgriebl/brickstore#5]
If you want to change the default currency it would automatically apply an exchange rate that you can't modify.

But if you open a file and change the currency of it from the bottom right, there it lets you chose between the exchange rate or put your own, and it works good, but just to that file...

**Comment by paramecie** [comment]
Not sure we're talking about the same problem, but...

It has a Currency in the Options, but when you set up a different one then USD (like EUR), it still opens files showing USD at the bottom right?

It should also automatically refresh the exchange rate every time your launch the application for example.
With maybe a warning if it can't get the rate like "Warning, using the previous rate, BrickStore couldn't refresh to the current one..."

**Comment by ZZJHONS** [comment]
That is another issue too, but not the same I described.

**Comment by paramecie** [comment]
Sorry - I'm to open an issue for this specific one.

**Comment by rgriebl** [comment]
Another round of cleaning up old issues... 
BS will always use the current ECB rates (updated daily) for any implicit conversions (e.g. the price guide). For explicit conversion (documents), it will also *default* to that rate, but it also gives you an option to adjust, because the file might be quite old and conversion would only make sense using an historical exchange rate.
TL;DR: the exchange rate shown in the settings dialog is just a hint to see today's ECB rate.

And yes, add a field to enter a custom rate for those implicit conversions would be possible, but would it make sense from a UX point of view? I can see a number of bug-reports "the price guide is totally off" coming in, if people would be allowed to set a custom rate.

Do you have a use-case in mind, where not using the daily rate would be beneficial?
